include/rocksdb/env.h

```diff
@@ -740,6 +740,14 @@ class RandomAccessFile {
   // RandomAccessFileWrapper too.
 };
 
+// A data structure brings the data verification information, which is
+// used togther with data being written to a file.
+struct DataVerificationInfo {
+  DataVerificationInfo(const std::string& checksum_) : checksum(checksum_) {}
+  // checksum of the data being written.
+  Slice checksum;
```

Comment:
I don't have a strong preference but think about pros and cons between `const Slice&` and `Slice`.

Reply:
What I'm thinking is the member of `DataVerificationInfo` can be changed (reuse of the variable). If the struct member is `const`, we are not able to change it once the `DataVerificationInfo` variable is constructed. And yes, the constructor is not necessary here.